ble/UUID.h

```diff
@@ -42,7 +97,7 @@ class UUID {
      *         different service or characteristics on the BLE device.
      *
      * @param longUUID
-     *          The 128-bit (16-byte) UUID value, MSB first (big-endian).
+     *          The 128-bit (16-byte) UUID value, LSB first (little-endian).
```

Comment:
Why not add an extra parameter in the constructor which allow the user to specify in which byte order the data he pass are ? 

```c++
UUID(const LongUUIDBytes_t longUUID, ByteOrder = BIG_ENDIAN)
```

The user shouldn't care about internal representation and should never use `getBaseUUID` for comparison. The comparison operator is here for that.